### PR TITLE
Add first half of front page

### PR DESCRIPTION
**Pull request by JamesMilnerUK**
## Description

This is the first half of the new Front Page. It takes inspiration from the original with some style changes. It is also responsive!

### Code

This PR touches:

- [ ] Content
- [x] Frontend
- [ ] Infrastructure
- [ ] Integration Tests

### Tests

- [x] Tests are included?

### Screenshots

screenshot_20190211_234739

screenshot_20190211_234752

dojo

responsive

### CODE REVIEW

File: src/App.m.css [hunk 1]
```diff
@@ -6,5 +6,5 @@
 .content {
 	flex-grow: 1;
 	display: flex;
-	margin-top: 80px;
+	margin-top: 78px;
```
Comment (KaneFreeman) [inline src/App.m.css]: Why 78 vs 80? If we use 80 we can use `--grid-size`

File: src/pages/Home.m.css [hunk 2]
```diff
@@ -1,3 +1,242 @@
...
+	text-transform: uppercase;
+}
+
+.build:hover {
+	font-weight: bold;
+	cursor: pointer;
+}
+
+.hero {
+	background-image: url('http://localhost:3000/herobg.27UmsJ1S.png');
```
Comment (KaneFreeman) [inline src/pages/Home.m.css]: URL to fix

File: src/App.m.css (hunk 1 above)
Comment (JamesMilnerUK) [reply src/App.m.css]: I'm going to use a unified `--header-size` var :+1:

File: src/pages/Home.m.css (hunk 2 above)
Comment (JamesMilnerUK) [reply src/pages/Home.m.css]: Deleted this

File: src/pages/Home.tsx [hunk 3]
```diff
@@ -1,13 +1,113 @@
...
+						Getting started with Dojo is simple. You can use your command line of choice and{' '}
+						<a href="https://www.npmjs.com">npm</a> to get going quickly.{' '}
+					</p>
+					<div key="cli" classes={[css.cli]}>
+						<Card dark={true} extraClasses={{ root: css.commands }}>
+							<div classes={css.command}>
+								<span classes={[css.commandOne, play ? css.commandOneAnimation : null]}>npm i @dojo/cli @dojo/cli-create-app -g</span>
+								<span classes={[play ? css.blinkOne : null]}>|</span>
+							</div>
+							<div classes={css.command}>
```
Comment (agubler) [inline src/pages/Home.tsx]: Are we going to be consistent with using arrays with vnode class props?

Comment (JamesMilnerUK) [reply src/pages/Home.tsx]: Yes good shout. We probably need to do this everywhere at some point.

**Pull request merged**